exercises/concept/remote-control-car/.docs/instructions.md

```diff
@@ -49,6 +49,8 @@ RemoteControlCar.display_distance(car)
 # => "0 meters"
 ```
 
+Make sure the function only accepts a `RemoteControlCar` struct as the argument.
+
```

Comment:
There is a test case for this requirement, but it wasn't mentioned in the instructions which was a bit annoying.